### PR TITLE
(#76) Term and phonetic spacing

### LINKED ISSUE #74: The audio icon wrapping issue for longer terms at smaller break points
### Issue description
*Description of the issue*
We are noticing issues with wrapping  of audio icons for longer terms such as "accelerated phase chronic myelogenous leukemia".  This is happening in all browsers.
> **ESTIMATE** 20

### Steps to reproduce the issue

1.  Perform SWS with "accelerated phase chronic myelogenous leukemia"

### What's the expected result?
The wrapping of audio icons should match with Prod at  smaller breakpoints.

### What's the actual result?
-The wrapping of audio icons does not match with Prod at  smaller breakpoints.

### Additional details / screenshot

image

### Related Tickets
* Issue #9999
* Issue #9999

### LINKED ISSUE #76: The spacing issue between the searched term and the phonetic spelling on the definitions in Search results page
### Issue description
*Description of the issue*
 The spacing should be tighter between the searched term and the phonetic spelling on the definitions in Search results page.

> **ESTIMATE** 20

### Steps to reproduce the issue

1.  Perform search with "mantle cell lymphoma"

### What's the expected result?
-  The spacing should be tighter between the searched term and the phonetic spelling on the definitions

### What's the actual result?
-Extra space exists between the searched term and the phonetic spelling on the definitions

### Additional details / screenshot

image

### Related Tickets
* Issue #9999
* Issue #9999

### PR DESCRIPTION
**Pull request by seyilonge-nci**
Closes #74
Closes #76
    
    * Updated DOM structure for pronunciation and styling
    * Reverted audio outline
    * Switched placement of styles
    * Inline display for audio and key

### CODE REVIEW

**Comment by seyilonge-nci** [comment]
@blilianyu @mworrest 
Please review https://react-app-dev.cancer.gov/sitewide-search-app/pr-82/?cfg=0&swkeyword=mantle%20cell%20lymphoma

File: src/components/atomic/audio-player/audio-player.scss [hunk 1]
```diff
@@ -7,6 +7,7 @@
 	position: relative;
 	width: 42px;
 	border: 0;
+	outline: none;
```
Comment (arcepaul) [inline src/components/atomic/audio-player/audio-player.scss:10]: omg.  no.  
Huge accessibility red flag.  How do I know where I am on the screen?

Comment (seyilonge-nci) [reply src/components/atomic/audio-player/audio-player.scss]: That actually wasn't intended to go in, needed it for testing. I've reverted!

**Comment by blilianyu** [comment]
Looks good! Passing!

**Pull request merged**